frontend/libs/health-department/feature-case-detail/src/lib/case-detail/case-detail.component.html

```diff
@@ -126,6 +126,14 @@
           [caseName]="caseDetail.firstName + ' ' + caseDetail.lastName"
         ></qro-contact-list>
       </mat-tab>
+      <mat-tab>
```

Comment:
Hinweis: In dem PR von Sebastian werden die einzelnen Tabs auf eigene Routen gelegt, die ihre Daten dann selbst laden. Das muss hier dann noch angepasst werden